### PR TITLE
updated tagalog translations in django.po

### PR DESCRIPTION
**Pull request by Zeroh729**
-Sir Nathan's edit

### CODE REVIEW

**Comment by jonathan-j-lee** [comment]
Hello! Thanks for the translations--we were about to open up a branch on our end to incorporate what Nathaniel sent us in his email.

Unfortunately, it looks like the build is failing right now because of syntax issues in the translations (more specifically, the placeholders for numbers), so the changes can't be merged yet. Today, I wrote up a short guide on how the [message file syntax](https://github.com/BerkeleyAutomation/malasakit-v1/wiki/Translation#messsage-file-syntax) works. 

Going to take a closer look at what changes need to be made.

File: malasakit-django/locale/tl/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -118,28 +126,28 @@ msgid ""
...
+msgstr "Maligayang pagdating"
 
 #: pcari/templates/landing.html:19
 #, python-format
 msgid ""
 "How prepared are you in times of disaster? %(num_responses)s barangay "
 "citizens have responded. Join them!"
 msgstr ""
-"Gaano ka kahanda sa panahon ng kalamidad? %(num_responses)s na ang nag "
+"Gaano ka kahanda sa panahon ng kalamidad? %(num_responses) na ang nag"
```
Comment (jonathan-j-lee) [inline malasakit-django/locale/tl/LC_MESSAGES/django.po]: Please add `s` back to the end of the format specifier `%(num_responses)`, just like for the English text (the `s` stands for a "string" placeholder).

File: malasakit-django/locale/tl/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -148,39 +156,42 @@ msgstr "Simulan"
...
 "Below, you can see how the other respondents answered the quantitative "
 "questions."
 msgstr ""
+"Makikita sa ibaba kung paano sinagutan ng iba ang mga kuwantitatibong katanungan."
 
 #: pcari/templates/peer-responses.html:42
 #, python-format
 msgid "<strong>%(num_ratings)s</strong> other people answered this question:"
-msgstr ""
+msgstr "<strong>%(num_ratings)</strong> katao na ang sumagot sa tanong na ito:"
```
Comment (jonathan-j-lee) [inline malasakit-django/locale/tl/LC_MESSAGES/django.po]: Please add `s` to the end of `%(num_ratings)`, just like for the English text.

File: malasakit-django/locale/tl/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -148,39 +156,42 @@ msgstr "Simulan"
...
 
 #: pcari/templates/peer-responses.html:50
 #, python-format
 msgid "Your answer: <strong id=\"answer-%(id)s\">None</strong>"
-msgstr ""
+msgstr "Ang iyong sagot: <strong id=\"sagot-%(id)\">Wala</strong>"
```
Comment (jonathan-j-lee) [inline malasakit-django/locale/tl/LC_MESSAGES/django.po]: The HTML code inside the pairs of angle brackets (`<...>`) should not be translated, so please change `sagot-%(id)` back to `answer-%(id)s`.

File: malasakit-django/locale/tl/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -148,39 +156,42 @@ msgstr "Simulan"
...
 
 #: pcari/templates/peer-responses.html:55
 #, python-format
 msgid "Average answer: <strong>%(mean)s</strong>"
-msgstr ""
+msgstr "Madalas na sagot ng iba: <strong>%(mean)</strong>"
```
Comment (jonathan-j-lee) [inline malasakit-django/locale/tl/LC_MESSAGES/django.po]: Please add `s` back to the end of `%(mean)`, just like for the English text.

**Comment by Zeroh729** [comment]
Hi! Thanks for checking. I've just pushed the changes.

File: malasakit-django/locale/tl/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -148,39 +156,42 @@ msgstr "Simulan"
...
 #, python-format
 msgid "<strong>%(num_ratings)s</strong> other people answered this question:"
-msgstr ""
+msgstr "<strong>%(num_ratings)s</strong> katao na ang sumagot sa tanong na ito:"
 
 #: pcari/templates/peer-responses.html:50
 #, python-format
 msgid "Your answer: <strong id=\"answer-%(id)s\">None</strong>"
-msgstr ""
+msgstr "Ang iyong sagot: <strong id=\"answer-%(id)\">Wala</strong>"
```
Comment (jonathan-j-lee) [inline malasakit-django/locale/tl/LC_MESSAGES/django.po]: There should be an `s` after the `%(id)` specifier (`answer-%(id)s` instead of `answer-%(id)`).

**Comment by jonathan-j-lee** [comment]
Looks good to me.

**Pull request merged**